spec/models/web_url_spec.rb

```diff
@@ -1,8 +1,17 @@
-require 'rails_helper'
+# frozen_string_literal: true
+
+require "rails_helper"
 
 RSpec.describe WebUrl, type: :model do
-  it { should belong_to(:web_urlable) }
+  it { is_expected.to belong_to(:web_urlable) }
+  it { is_expected.to validate_presence_of(:url) }
 
+  describe "#url" do
+    it "only accepts valid urls" do
```

Comment:
Bei der Beschreibung hätte ich  einen `.to be_valid` Test erwartet. Kann das umformuliert werden?
Und fehlt dann nicht noch ein `.to be_valid` Test für den positiven Fall?

Reply:
Ja das schaue ich mir nochmal genauer an, wenn ich das nchste mal specs schreibe